pkg/certificates/constants.go

```diff
@@ -33,3 +35,14 @@ const (
 	SecretCertKey   = "public-cert.pem"
 	SecretPKKey     = "private-key.pem"
 )
+
+func DataPlaneRoutingName(rountingId string) string {
+	if rountingId == "" {
+		rountingId = "0"
+	}
+	return dataPlaneRoutingPrefix + rountingId
```

Comment:
What is the format of the expected id here? From what I read this is read from a label abd used to create a san and compare with the cert side. When we validate secrets later on we do:

`	for _, san := range sans {
		for _, certSan := range cert.DNSNames {
			if certSan == san {
				continue OUTER
			}
		}
		return nil, nil, fmt.Errorf("missing san %s", san)
	}`

However the comparison should be [case-insensitive ](https://security.stackexchange.com/questions/150770/are-certificate-sans-case-sensitive) right?